proto/success_rate.proto

```diff
@@ -64,5 +68,9 @@ message InvalidateWindowsRequest {
 }
 
 message InvalidateWindowsResponse {
-    string message = 1;
+   enum InvalidationStatus {
```

Comment:
nit: why not have a unified status that we can use for app successes and failures?

Reply:
ya we can do that, will create an issue for it